Judicial Process Changes (#51)
funky-station/docs: src/design-proposals/judicial-process.md

# Changes to the Judicial Process

| Designers | Implemented | GitHub Links |
|---|---|---|
| The Boss 9 | :x: No | PR Links or TBD |

## Overview

> This Doc puts a recent discussion into writing. Instead of Warden being the one who convicts and sentences, the Magistrate will be the one who determines what someone is guilty of, and what length their sentence is. And of course it will take place in the Courtroom

Reviewer: ```suggestion
This document regards a major change to the judicial process recently discussed in the discord server. In short, the Warden will be removed from their responsibilities of determining guilt and sentencing, with such responsibilities wholly being moved to the Magistrate.
```

## Background

Currently the amount of Court and Law related shenanigans are too low. Lawyers and Magistrates would like some more attention, and honestly the Warden having 100% of the control over sentencing feels a little lacking RP wise

## Features to be added

> Okay so when Security arrests someone the Inmate will be transported to the Processing room as usual. They will be searched for any contraband, and any personal items will be left in the processing room. Then the Inmate will be escorted to the courtroom. 

Reviewer: ```suggestion
When Security arrests a suspect, that suspect will be moved to Security for processing. During processing, the suspect is searched for and removed of any contraband they have on their person. Additionally, any personal effects are stored in processing for later collection. After processing is complete, the suspect is escorted to the courtroom for sentencing.
```

> In the Courtroom guilt will be determined. Any evidence of the crime will be brought to the Courtroom by the escorting officer. Any witnesses or victims that were related to the crime that are willing to testify should be in the court too. The Magistrate will then proceed with a hasty trial to determine sentence and guilt.

Reviewer: ```suggestion
Sentencing belongs in the courtroom, where guilt will be determined and sentencing made by the Magistrate. All forms of evidence are brought to the courtroom by the escorting officer, with any witnesses or victims related to the crime able to testify within the courtroom. As a note, any such testifying witnesses should be in the courtroom before the case is judged. The Magistrate should prioritize speed during their trial, through which they will determine if the suspect is guilty - and if so, what the sentence should be.
```

The Inmate will be escorted back to processing where the Warden simply finishes locking up their personal items and gives them their Prisoner ID.

Now, for actual changes needed to take place

> Both the Courtroom and the Processing room PLEASE need a Wanted Console. This is so that the Warden and the Magistrate can communicate crucial information about Inmates to everyone that needs to know.

Reviewer: ```suggestion
The courtroom and the processing room need to have a criminal records computer such that the Warden and Magistrate can communicate information regarding suspects to all necessary parties.
```

> The "Detained" status needs to have a description. For "Wanted" people, a description can already be added. For "Detained" people, the option to add a description is crucial here. This is so that when the Magistrate sentences an inmate, they can mark them as "Detained" and mark how much time they should be put in Brig for.

Reviewer: ```suggestion
The "Detained" status needs to be able to be applied with a description, similar to the way the "Wanted" status can. This is so that when the Magistrate sentences a suspect, the Magistrate can mark them as "Detained" and mark them for their sentence length.
```

A new status needs to be added called "Awaiting Trial". This will be a status given to inmates that have been arrested. The Magistrate will be able to instantly see that their presence in Court is needed, which is necessary.

> SOP Changes!

Reviewer: ```suggestion
### Regarding changes to SOP
```

> Some minor SOP changes need to happen so that everyone knows their role in these new trials.

Reviewer: ```suggestion
SOP needs to be edited such that all relevant parties know what their roles are in this new judicial process.
```

Space Law should change, and all crimes should be ~doubled in length, so that these trials aren't occuring just for a 2 minute sentence lol. Should be good RP, which hopefully makes the longer waits feel worth it.

If the Magistrate is absent, then the NTR will substitute in for them. If the NTR and Magistrate are absent, then there will be no Court. I'm not going to ask the Captain to be present for every Court Case, so unless the Captain has free time and is willing to be the Judge, then the Judicial Process will halt and the system will just go back to the Warden sentencing Inmates in the Processing room.

> In the event of a Red Alert, or a known threat worthy of a Red Alert is present, then the Judicial Process is halted. The Warden will sentence people in the Processing room like before.

Reviewer: ```suggestion
If the station is on red alert, or the station is otherwise facing a station-ending threat, this judicial process is halted. As a fallback, we will use what we currently use, with the Warden judging and sentencing suspects in Security.
```

A new system of Trials will probably have to be added, designed to simply establish guilt and sentence without taking time for irrelevant matters.

## Game Design Rationale

- This Judicial Process system will add many new ways to RP, and should bring consistent RP to lawyers and to CentComm personell. It should decrease the workload of the Warden aswell, and will make sentencing feel more RP'ing, instead of just rushing through sentence after sentence. Trials will be cool again, trust

## Roundflow & Player interaction

- This feature will be present in every round, since there is always Inmates. I want people that witness crimes and are victims to crimes to show up to Court on their own accord. Both to defend the Inmate and to Prosecute the Inmate. 

> Magistrates should make sure they're communicating with Lawyers and IAAs since both of them usually like trials.

Reviewer: ```suggestion
Magistrates will need to communicate with greater frequency to lawyers and IAAs due to their relevance in defending and prosecuting suspects.
```

> And the option to halt the Judicial Process is an option if Security is busy with greater threats, so it shouldn't ever be bothersome to attend Court as Security or CC.

Reviewer: ```suggestion
The option to halt the Judicial Process system and fallback to the sentencing system we currently use is useful if Security is busy fighting station-ending threats - it should never be bothersome to attend court as Security or Centcomm VIPs.
```

## Administrative & Server Rule Impact 

- This rule shouldn't require any additional admin intervention. People are already getting sentenced, this is just a more formal way of doing it
- I don't think this feature will add more potential for griefing. When Court becomes more frequent, there shouldn't be any Court bombers or anything

> - The rules will just be enforced by the SOP mostly

Reviewer: ```suggestion
- These judicial processes will mostly be enforced by SOP.
```

# Technical Considerations

> - As stated before, some Mapping Changes are required to add Wanted Consoles to the Processing room and the Courtroom

Reviewer: ```suggestion
- Mapping changes will be necessary to the extent that criminal records computers are added to processing areas and courtrooms.
```

> - Also as stated before, some of the status's need to be changed or added to make communication easier concerning Inmates

Reviewer: ```suggestion
- Some statuses placed via the criminal records computer will need to be changed and added to make communication regarding suspects and trials more convenient.
```
